docs/en/08_troubleshooting.md

```diff
@@ -95,3 +95,113 @@ _-: php: command not found
 This restriction is a security feature coming with Plesk 10.
 On round about page 150 of the plesk Administrator Guide you will find a solution to enable scheduled tasks which use the command line. (But the latest Guide for 10.3.1 mentions `/usr/local/psa/admin/bin/server_pref -u -crontab-secure-shell "/bin/sh"` although "server_pref" doesnt exit.
 Since we are using a dedicated server for only one customer, we defined the crons under "Server Management"->"Tools & Utilities"->"Scheduled Tasks"->"root". The security restrictions of plesk are not involved then.
+
+## Broken jobs
+
+Sometimes, jobs break without having any issues with their implementation but rather an external factor is the root cause, for example database table lock may prevent a DB write.
+Most common scenario is "publish" action related jobs such as scheduled publish feature which may experience DB deadlocks on the versioned table as this can be frequently accessed.
+
+For this scenario it's recommended to configure automatic job retries.
+Configuration has some flexibility on number of retries and the timing of the retry attempts.
+
+### Basic configuration
+
+This configuration is recommended as a good starting point when trying to set up automatic retries.
+
+* `max_retry_attempts` - number of retry attempts, this allows to control how many times a broken job is retried
+* `initial_retry_delay` - minimal waiting time before the retry attempt is executed, this helps spread retry attempts apart from each other
+
+This configuration is applied to your job class.
+
+### Advanced configuration
+
+In case you have specific scenarios that can't be quite covered by basic configuration you can use the advanced configuration which provides more control over the automated job retries.
+
+Use the sample configuration below as a starting point and adjust as needed.
+This code snippet need to be placed into your job class.
+
+* `retry_falloff_multiplier` provides the capability to increase the retry period with each retry attempt, defaults to `1`
+* `retry_falloff_multiplier_variance` acts as a modifier for `retry_falloff_multiplier`, needs to be always lower value compared to `retry_falloff_multiplier`, this allows you to break up clusters of broken jobs which can prevent load spikes and DB deadlocks from forming, defaults to `0`
+
+**Examples**
+
+```php
+// Linear retry pattern
+private static int $max_retry_attempts = 4;
```

Comment:
FYI I suspect the thing causing docs linting to fail is syntax failure here - since these aren't in a class, they're not valid syntax (you can't have a private static variable in the global scope).

If you wrap it in a class I think it'll pass, e.g
```php
namespace App\Jobs;

use Symbiote\QueuedJobs\Services\AbstractQueuedJob;

class MyJob extends AbstractQueuedJob
{
    // the examples all go here
    // ...
}
```

Note that the `// ...` should be kept - we use that in a lot of the dev docs as a shorthand for "there's more to the class it's just not relevant for this code snippet"